src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +82,11 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!(
+              title.length !== 0
+            && imdbId.length !== 0
```

Comment:
Empty string will be falsy itself
```suggestion
            && imdbId
```